sqlglot/dialects/doris.py

```diff
@@ -128,6 +130,37 @@ def _parse_partition_by_opt_range(
                 create_expressions=create_expressions,
             )
 
+        def _parse_build_property(self) -> exp.BuildProperty:
+            mode = self._parse_var(upper=True)
+            return self.expression(exp.BuildProperty, this=mode)
```

Comment:
[Nit] We tend to inline these:

```suggestion
            mode = self._parse_var(upper=True)
            return self.expression(exp.BuildProperty, this=self._parse_var(upper=True))
```